src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
```

Comment:
empty line between logic part of code (variables, methods ....)

```suggestion
    result: 0,

```